src/DynamicMvvm.Abstractions/DynamicMvvm.Abstractions.csproj

```diff
@@ -30,6 +31,10 @@
 	<ItemGroup>
 		<PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="3.1.0" />
 		<PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="3.1.0" />
+	</ItemGroup>
+
+	<ItemGroup>
```

Comment:
you probably don't need a separate ItemGroup

Reply:
(Same comment applies to other csproj files)

Reply:
I want to seperate the source link support to make it more obvious that it's an add-on